src/Raven.Server/Rachis/RachisConsensus.cs

```diff
@@ -1489,6 +1491,17 @@ public long AppendToLog(CommandBase cmd, long term)
                 GetLastTruncated(context, out lastIndex, out long _);
             }
             lastIndex += 1;
+            var guid = LogHistory.GetGuidFromCommand(cmd);
+            if (guid == RaftIdGenerator.DontCareId)
+            {
+                var newGuid = $"DontCare/{lastIndex}";
```

Comment:
after second thought, in order this to be unique the index isn't enough, it might happened that we will have same index on different terms, so let's add the term here as well.

Reply:
Done